### ISSUE TITLE
Why my pretrain loss doesn't decrease?

### ISSUE THREAD

**Issue by keqicmz** [935963004/LaBraM#63]
Hi authors,

Thank you for your excellent work! I have some problems during the pre-training phase of the training process. The loss did not show a significant decrease during the training process. Have you encountered similar problems during the training process? And i want to know which aspect should the investigation be conducted? 
Thank you very much.
